src/foam/nanos/crunch/Capability.js

```diff
@@ -88,6 +88,10 @@ foam.CLASS({
       name: 'keywords',
       class: 'StringArray'
     },
+    {
```

Comment:
Why is this property needed if there's a *:* relationship between Capability and CapabilityCategory?

Reply:
Oops, forgot to remove that